### PR TITLE
Change the interface of aggregate

### PR DESCRIPTION
**Pull request by tpoterba**

### CODE REVIEW

File: python/hail/matrixtable.py [hunk 1]
```diff
@@ -1400,52 +1400,47 @@ def aggregate_rows(self, **named_exprs):
...
         annotations = [ptypes[i]._convert_to_py(result_list[i]._1()) for i in range(len(ptypes))]
-        d = {k: v for k, v in zip(named_exprs.keys(), annotations)}
-        return Struct(**d)
+        return annotations[0]
 
     @handle_py4j
-    def aggregate_cols(self, **named_exprs):
-        """Aggregate over columns into a local struct.
+    def aggregate_cols(self, expr):
+        """Aggregate over rows to a local value.
```
Comment (patrick-schultz) [inline python/hail/matrixtable.py]: Aggregate over columns

File: python/hail/matrixtable.py [hunk 2]
```diff
@@ -1457,52 +1452,47 @@ def aggregate_cols(self, **named_exprs):
...
         annotations = [ptypes[i]._convert_to_py(result_list[i]._1()) for i in range(len(ptypes))]
-        d = {k: v for k, v in zip(named_exprs.keys(), annotations)}
-        return Struct(**d)
+        return annotations[0]
 
     @handle_py4j
-    def aggregate_entries(self, **named_exprs):
-        """Aggregate over all entries into a local struct.
+    def aggregate_entries(self, expr):
+        """Aggregate over rows to a local value.
```
Comment (patrick-schultz) [inline python/hail/matrixtable.py]: Aggregate over entries

**Comment by tpoterba** [comment]
@konradjk group_by/aggregate is doing something pretty different, so the interface there is totally what you want.

I'd be open to renaming these things `query` like they were, but I think Cotton has a strong preference for aggregate.

**Comment by konradjk** [comment]
Ok that's fine. May just want to make notes of that in the docs.

**Pull request merged**